R/sim.R

```diff
@@ -387,7 +387,7 @@ sim <- function (ode = NULL,
     p_i <- p
     if(!is.null(covariates_table)) {
       covariates_tmp <- covariates_table[[i]]
-      design_i <- create_event_table(regimen, t_max, t_obs, t_tte, t_init = t_init, p_i, covariates_table[[i]])
```

Comment:
unrelated, but noticed that this instance of calling create_event_table did not have all arguments, which could in specific cases lead to errors.